cloudinary_cli/modules/upload_dir.py

```diff
@@ -25,18 +25,27 @@
              "You can specify a whole path, for example folder1/folder2/folder3. "
```

Comment:
@const-cloudinary - Suggest changing this help text to: 

The path where you want to upload the assets.  The path you specify will be pre-pended to the public IDs of the uploaded assets.  You can specify a whole path, for example path1/path2/path3.